examples/create_libraries/lib/src/hw.dart

```diff
@@ -0,0 +1 @@
+String get message => 'Hello World!';
```

Comment:
This is the stub implementation, typically it's called `hw_stub.dart` and often it might be useful to throw an `Error`.
I would suggest [UnimplementedError](https://api.dartlang.org/stable/2.6.0/dart-core/UnimplementedError-class.html) because this is not implemented, and this file is not intended to be imported into any actual production code.

I suspect @lrhn would suggest throwing [UnsupportedError](https://api.dartlang.org/stable/2.6.0/dart-core/UnsupportedError-class.html)? :)

But throwing an `Error` from the `_stub.dart` implementation is probably a good pattern to give users. Sometimes a stub implementation can implement the full feature -- but often it's a bug if the stub implementation is imported.

Reply:
I'd say `UnsupportedError`, yes. If you plan to later implement, but you haven't gotten to it yet, you use `UnimplementedError`. It's really only something you should use during development. The `UnsupportedError` is for permanent lack of support, like writing to a read-only list.

Otherwise I agree. I would probably use `hw_none.dart` as name, but `stub` is fine too.